Library/Homebrew/extend/os/pkgconf.rb

```diff
@@ -0,0 +1,52 @@
+# typed: strict
+# frozen_string_literal: true
+
+module Homebrew
+  module Pkgconf
+    module_function
+
+    sig { returns(T.nilable(T::Hash[Symbol, String])) }
```

Comment:
```suggestion
    sig { returns(T.nilable([String, String])) }
```
or something might be nicer